CHANGELOG.rst

```diff
@@ -33,6 +32,7 @@ Internal changes:
 
 - Select the :option:`--html-theme` using CSS classes. (:issue:`650`)
 - Change and extend ``cmake`` tests. (:issue:`676`)
+- Detect the verion of the ``gcc``. (:issue:`686`)
```

Comment:
I'd provide context that this version detection only applies to tests, not to user-visible behaviour.

```suggestion
- Detect ``gcc`` version for running tests. (:issue:`686`)
```